### DISCUSSION TITLE
Unable to use TraceLoggingProvider etw_provider_handle ("ONNXRuntimeTraceLoggingProvider") in onnxruntime\core\session\onnxruntime_c_api.cc

### DISCUSSION THREAD

**Discussion by tzimmer420** [microsoft/onnxruntime#23147] [General]
### Describe the issue

Unable to use ONNXRuntimeTraceLoggingProvider in onnxruntime\core\session\onnxruntime_c_api.cc.   Build is unable to resolve "etw_provider_handle".

### To reproduce

Add code with TraceLoggingWrite(etw_provider_handle, ...) in onnxruntime_c_api.cc.
Code will not build.

### Platform

Windows

### OS Version

26100

### ONNX Runtime Installation

Built from Source

### ONNX Runtime Version or Commit ID

1.20.1

### ONNX Runtime API

Python

### Architecture

X64

### Execution Provider

Other / Unknown

**Comment by snnn** [comment]
It doesn't seem to be a bug of onnxruntime.  I converted it from Github issue to a discussion.
